terraform/prd/tokyo/rds.tf

```diff
@@ -0,0 +1,89 @@
+locals {
+  rds_engine         = "aurora-postgresql"
+  rds_engine_version = "15.3"
+  rds_cluster_identifier = "llm-ops-prd"
+}
+
+resource "aws_db_subnet_group" "rds_tokyo_db_subnet_group" {
+  provider   = aws.primary
+  name       = "dpsh-platform-prd-db-subnet-group"
+  subnet_ids = [data.aws_subnet.news_private_b_0.id, data.aws_subnet.news_private_c_0.id, data.aws_subnet.news_private_d_0.id]
+
+  tags = {
+    Cost   = var.cost
+    Env    = "production"
+    Name   = "dpsh-platform-prd"
+    Team   = var.team
+    System = var.system
+  }
+}
+
+resource "aws_rds_cluster" "rds_tokyo_cluster" {
+  provider                    = aws.primary
+  deletion_protection         = true
+  engine                      = local.rds_engine
+  engine_version              = local.rds_engine_version
+  db_subnet_group_name        = aws_db_subnet_group.rds_tokyo_db_subnet_group.name
+  vpc_security_group_ids      = [data.aws_security_group.smartnews.id]
+  cluster_identifier          = "llm-ops-prd"
+  master_username             = "postgres"
+  master_password             = "postgres"
```

Comment:
Create a secret manager to store the password like here: https://github.com/smartnews/DP-clickhouse/blob/master/clusters/terraform/module/main.tf#L30-L54 

And refer to the password from random_password.

Reply:
got it, for quick set up, let me create the secret later